### PR TITLE
import logging levels from logging for convenience

### PR DESCRIPTION
**Pull request by keflavich**
I find it awkward that you can import `log` (`from astropy import log`) and
have access to an awesome logger, but to set the levels, you either have to
separately `import logging` or memorize the numerical logging levels.

For example, now, if I want to enable debugging, I either do:
```
from astropy import log
import logging
log.setLevel(logging.DEBUG)
log.debug("message")
```
or

```
from astropy import log
log.setLevel(10)
log.debug("message")
```

This change imports the DEBUG, CRITICAL, WARN, INFO, FATAL, ERROR, and NOTSET
variables into `astropy.log`'s namespace

### CODE REVIEW

File: astropy/logger.py [hunk 1]
```diff
@@ -19,6 +19,14 @@
 
 __all__ = ['Conf', 'conf', 'log', 'AstropyLogger', 'LoggingError']
 
+# import the logging levels from logging so that one can do:
+# log.setLevel(log.DEBUG), for example
+logging_levels = ['DEBUG', 'CRITICAL', 'FATAL', 'ERROR', 'WARN', 'WARNING',
+                  'NOTSET', 'INFO']
+for level in logging_levels:
+    __import__('logging', level)
```
Comment (mdboom) [inline astropy/logger.py]: Alternatively (do avoid calling into `__import__` which can be slow), do:

```
globals()[level] = getattr(logging, level)
```

**Comment by mdboom** [comment]
Seems like a good idea to me.  Should have a CHANGELOG entry (technically not an API breakage, I guess, just an extension).

**Comment by embray** [comment]
:+1:

File: astropy/logger.py [hunk 2]
```diff
@@ -19,6 +19,14 @@
 
 __all__ = ['Conf', 'conf', 'log', 'AstropyLogger', 'LoggingError']
 
+# import the logging levels from logging so that one can do:
+# log.setLevel(log.DEBUG), for example
+logging_levels = ['DEBUG', 'CRITICAL', 'FATAL', 'ERROR', 'WARN', 'WARNING',
```
Comment (embray) [inline astropy/logger.py]: Let's not import WARN, and just use WARNING.  I think the fact that both exist is of historical interest, but as far as importing into Astropy goes I don't think we need both.  I think also you should list these in their numerical order, so DEBUG comes after INFO, etc.  Just a minor nit.

Comment (embray) [inline astropy/logger.py]: (As far as listing them in order goes I think it's helpful just for documentation purposes).

**Comment by keflavich** [comment]
@embray reordered.

I don't understand the sphinx build failure.  It seems to be related to configuration items I haven't touched:
https://travis-ci.org/astropy/astropy/jobs/70841289#L4182
```
/home/travis/build/astropy/astropy/docs/api/astropy.config.ConfigAlias.rst:15: WARNING: py:obj reference target not found: cfgtype
/home/travis/build/astropy/astropy/docs/api/astropy.config.ConfigItem.rst:15: WARNING: py:obj reference target not found: cfgtype
/home/travis/build/astropy/astropy/docs/api/astropy.utils.state.ScienceStateAlias.rst:1: WARNING: py:obj reference target not found: ConfigItem
/home/travis/build/astropy/astropy/docs/api/astropy.utils.state.ScienceStateAlias.rst:1: WARNING: py:obj reference target not found: ConfigItem
/home/travis/build/astropy/astropy/docs/api/astropy.utils.state.ScienceStateAlias.rst:1: WARNING: py:obj reference target not found: ConfigItem
/home/travis/build/astropy/astropy/docs/api/astropy.utils.state.ScienceStateAlias.rst:9: WARNING: py:obj reference target not found: ConfigItem
/home/travis/build/astropy/astropy/docs/api/astropy.utils.state.ScienceStateAlias.rst:15: WARNING: py:obj reference target not found: cfgtype
/home/travis/build/astropy/astropy/docs/api/astropy.utils.state.ScienceStateAlias.rst:9: WARNING: py:obj reference target not found: ConfigItem
/home/travis/build/astropy/astropy/docs/api/astropy.utils.state.ScienceStateAlias.rst:9: WARNING: py:obj reference target not found: ConfigItem
/home/travis/build/astropy/astropy/docs/api/astropy.utils.state.ScienceStateAlias.rst:24: WARNING: py:obj reference target not found: ConfigItem
```
though I may not understand how to search sphinx warnings correctly.

**Comment by mdboom** [comment]
I think the Sphinx errors are unrelated, but were on master until  10ceca1cadc17e1bb60f2357c207fd57d7c6c710

**Comment by embray** [comment]
:+1:

**Comment by astrofrog** [comment]
Note that the reason I never came across this is because ``setLevel`` can also take a string, and I always use that :)

```
In [4]: from astropy import log

In [5]: log.setLevel("INFO")
```

**Pull request merged**